newIDE/app/src/ProjectCreation/LocalExamples.js

```diff
@@ -22,20 +22,6 @@ type Props = {|
   onCreateFromExampleShortHeader: OnCreateFromExampleShortHeaderFunction,
 |};
 
-export const showGameFileCreationError = (
-  i18n: I18nType,
-  outputPath: string,
-  rawError: Error
-) => {
-  showErrorBox({
-    message: i18n._(
-      t`Unable to create the game in the specified folder. Check that you have permissions to write in this folder: ${outputPath} or choose another folder.`
-    ),
-    rawError,
-    errorId: 'local-example-creation-error',
-  });
-};
-
 export default function LocalExamples({
```

Comment:
If you're not importing `LocalExamples` or `BrowserExamples` anymore, are they still useful?

Reply:
I forgot to remove them, thanks!